packages/@aws-cdk/aws-lambda/lib/event-source-mapping.ts

```diff
@@ -91,6 +91,14 @@ export interface EventSourceMappingOptions {
    * @default 1
    */
   readonly parallelizationFactor?: number;
+
+  /**
+   * The name of the Kafka topic.
+   * Maximum of 1
+   *
+   * @default - no topics
+   */
+  readonly topics?: string[];
```

Comment:
Make this a scalar since it can't be more than 1.

Reply:
Probably nice to name this `kafkaTopic` as well.

Reply:
Thanks for the feedback @nija-at, I've addressed the comments on https://github.com/aws/aws-cdk/pull/10445/commits/c698a2951646628049532d1212b9ffd2c738865c